pkg/publish/ingress.go

```diff
@@ -95,7 +102,7 @@ func Ingress(req router.Request, app *v1.AppInstance) (result []kclient.Object,
 			}
 			for _, domain := range cfg.ClusterDomains {
 				hostPrefix := toPrefix(domain, svcName, app)
-				hostname := hostPrefix + domain
+				hostname := name.Limit(hostPrefix+domain, 255)
```

Comment:
we cant do this. it wold lop off the end of the domain name

Reply:
you would have to trim host prefix, but i dont think its important to do...we are pretty much garaunteed to not violate the 255 (for now)